.cargo/config.toml

```diff
@@ -10,7 +10,7 @@ rustflags = [
   "-Aclippy::all",
   "-Dclippy::correctness",
   "-Aclippy::if-same-then-else",
-  "-Aclippy::clone-double-ref",
+  "-Asuspicious_double_ref_op",
```

Comment:
Just a note: this is a drive-by update unrelated to the rest of the PR. Without this change a useless warning is emitted for many packages.